### DISCUSSION TITLE
Theme switching not supported on my desktop environment

### DISCUSSION THREAD

**Discussion by Deewens** [oskarsh/Yin-Yang#271] [Q&A]
Hi,
I installed Yin-Yang from source on Ubuntu 22.04.4 LTS. After installation, I had an issue with libxcb-cursor library that was not installed, so the app did not launch, so I just apt install it. Now, I can launch Yin Yang, but it just won't work. It gives me the following logs in the terminal:
```bash
Plugin System has no support for your desktop environment yet!
Plugin Colors has no support for your desktop environment yet!
Plugin GTK has no support for your desktop environment yet!
Plugin Icons has no support for your desktop environment yet!
Plugin Wallpaper has no support for your desktop environment yet!
Error while loading translation. Using default language.
```

Thanks for your help

**Comment by l0drex** [comment]
The logs don't indicate a critical problem. What exactly is not working? Does it not launch, does the theme change not work, …

**Comment by Deewens** [comment]
Sorry, I forgot to keep track of this discussion, I had other issue with Ubuntu and ended up replacing it with Fedora.
